packages/grafana-ui/src/components/Text/Text.tsx

```diff
@@ -22,26 +33,78 @@ export interface TextProps {
   italic?: boolean;
   /** Whether to align the text to left, center or right */
   textAlignment?: CSSProperties['textAlign'];
-  children: React.ReactNode;
+  children: NonNullable<React.ReactNode>;
 }
 
 export const Text = React.forwardRef<HTMLElement, TextProps>(
-  ({ element = 'span', variant, weight, color, truncate, italic, textAlignment, children }, ref) => {
+  ({ element = 'span', variant, weight, color, truncate = false, italic, textAlignment, children }, ref) => {
```

Comment:
Is there a reason to set this to `false` instead of just not setting a default which will be falsy as well?